### PR TITLE
feat(frontend): implement BTC txs components

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

In this PR, I implemented components needed to display BTC transactions. In a separate PR, the new `$lib/components/transactions/Transaction.svelte` component will be reused by `EthTransaction.svelte` (similar like it's done in `BtcTransaction.svelte`).

### CODE REVIEW

File: src/frontend/src/btc/components/transactions/BtcTransactions.svelte [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import { slide } from 'svelte/transition';
+	import BtcTransaction from '$btc/components/transactions/BtcTransaction.svelte';
+	import {
+		sortedTransactions,
+		transactionsNotInitialized
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/transactions/BtcTransactions.svelte:6]: In another PR can you prefix those derived store with `btc`?

We got now two derived stores called `transactionsNotInitialized` within the all app.

File: src/frontend/src/lib/components/transactions/Transaction.svelte [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	$: icon = type === 'send' ? IconSend : IconReceive;
+</script>
+
+<button class="contents">
+	<Card>
+		{label}
+
+		<RoundedIcon slot="icon" {icon} iconStyleClass={pending ? 'opacity-10' : ''} />
+
+		{#if nonNullish(value)}
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/transactions/Transaction.svelte:33]: This works? Isn't Svelte complaining? Should you use a `<svelte:fragment slot="amount">` and have the `if` finding place within the fragment?

File: src/frontend/src/lib/components/transactions/Transaction.svelte [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	let label: string;
+	$: label = type === 'send' ? $i18n.send.text.send : $i18n.receive.text.receive;
+
+	let icon: ComponentType;
+	$: icon = type === 'send' ? IconSend : IconReceive;
+</script>
+
+<button class="contents">
+	<Card>
+		{label}
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/transactions/Transaction.svelte:29]: In `IcTransaction` the label is wrapped within a span, would it be interesting here?

Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/transactions/Transaction.svelte:29]: Yes! Added `span` with the CSS classes as in IcTransaction.

File: src/frontend/src/lib/components/transactions/Transaction.svelte (hunk 2 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/components/transactions/Transaction.svelte:33]: Good catch, it wasn't displaying properly. Fixed it.

File: src/frontend/src/btc/components/transactions/BtcTransactions.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/transactions/BtcTransactions.svelte:6]: Sure, noted.

File: src/frontend/src/btc/components/transactions/BtcTransaction.svelte [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+	import { BigNumber } from '@ethersproject/bignumber';
+	import type { BtcTransactionStatus, BtcTransactionUi } from '$btc/types/btc';
+	import Transaction from '$lib/components/transactions/Transaction.svelte';
+	import { modalStore } from '$lib/stores/modal.store';
+	import type { TransactionType } from '$lib/types/transaction';
+
+	export let transaction: BtcTransactionUi;
+
+	let value: bigint | undefined;
+	let timestamp: number | undefined;
```
[the remark this answers is not in the document]
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/transactions/BtcTransaction.svelte:12]: Yes, it's a number in the Blockchain API, so I used it like this in the UI type too.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**